### ISSUE TITLE
Question about TOP1 Loss

### ISSUE THREAD

**Issue by sungjun3010** [yhs-968/pyGRU4REC#9]
Hi, thank you for your work.

Here I have a question for your top-1 loss implementation.

At `modules/loss.py: line 56`
loss = F.sigmoid(diff).mean() + *F.sigmoid(logit ** 2).mean()*

If I understood top-1 loss and your code, `logit` have already contained positive sample, but *F.sigmoid(logit ** 2).mean()* should be applied for only negative samples

Could you please check whether I understand right? 

Thanks!
